docs/audit-log-filter/audit-log-filter-overview.md

```diff
@@ -0,0 +1,127 @@
+# Audit Log Filter overview
+
+The Audit Log Filter plugin allows you to monitor, log, and block a connection or query actively executed on the selected server. 
+
+Enabling the plugin produces a log file that contains a record of server activity. The log file has information on connections and databases accessed by that connection. 
+
+The plugin uses the `mysql` system database to store filter and user account data. Set the [`audit_log_database`](audit-log-filter-options.md#audit_log_database) variable at server startup to select a different database.
+
+The [`AUDIT_ADMIN`](audit-log-privileges.md) privilege is required to enable users to manage the Audit Log Filter plugin.
+
+## Privileges
```

Comment:
We could mention AUDIT_ABORT_EXEMPT privilege in this section as well - https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_audit-abort-exempt

Reply:
Done.